_posts/2015-11-24-hoodie-updates.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Hoodie 1.0 milestones closing in"
+author: Jan
+layout: blog-post-tgif
+comments: true
+---
+
+This is Part 2 of of our [little series about what has happened in Hoodie-land](http://hood.ie/blog/we-have-been-busy.html "We. Have. Been. Busy!") in the past <strike>three</strike>five months.
+
+Big news, big news: we’re getting there! After [a very busy summer](prev blog post) we are closing in on the final tasks for [the Hoodie 1.0 milestones](http://gr2m.github.io/milestones/).
```

Comment:
Link to prev blog post to insert on this line: http://hood.ie/blog/we-have-been-busy.html